modules/thekla_unwrap/config.py

```diff
@@ -3,5 +3,5 @@ def can_build(platform):
 
 def configure(env):
     if not env['tools']:
-        env['module_thekla_unwrap_enabled'] = False
+        env['builtin_thekla_atlas'] = False
```

Comment:
Hm no, that was better before. `builtin_thekla_atlas` is meant to be used as an option to use a system-provided shared library. For this, it's likely not relevant at all in the first place as I don't think any distro will package thekla_atlas.